kitsune/wiki/templates/wiki/review_revision.html

```diff
@@ -16,6 +16,23 @@
       {{ _('Revision Comment: {comment}')|f(comment=revision.comment) }}
       </p>
 
+      {% if unreviewed_revision %}
+        <p>
+        <div class="unreviewed-revision"> {{ ngettext('Unreviewed Revision:', 'Unreviewed Revisions:', unreviewed_revision|length ) }} </div><br>
+        <ul class="revision-comment">
+          {% for revision in unreviewed_revision %}
+            <li>
+              {{ _('Revision {id} by {user}')|f(id=revision.id, user=revision.creator) }}.
+              <br>
+              {{ _('<label>Revision Comment:</label> {comment}')|f(comment=revision.comment) }}
+              <br>
+              <a href="{{ url('wiki.review_revision', document.slug, revision.id) }}">{{ _('Review Revision {id}')|f(id=revision.id) }}</a>
+            </li>
+        {% endfor %}
```

Comment:
The indentation here does not match the indentation of the opening `{% for %}` tag, on line 23.